User-Help/GlobalTrustLevels.md

```diff
@@ -0,0 +1,138 @@
+| **Access to all other moderation tools**  | n/a                                                                                  | n/a                                                                                                  | n/a                                              | n/a                                                                                                 | n/a                                                                               | Full access                                                    |
+
+
+
+# Glossary of terms
+
+### Prerequisites
+
+Minimum activity required to reach a given trust level.
+
+### Posts
+
+Typically posts are questions. Depending on the category, however, they can also include other formats like blogs or wikis.
+
+### Responses 
+
+A response is a standalone reply to a post, typically a fully researched answer to a question. It may be something else depending on category, e.g. a submission to a contest. 
+
+### Comments
+
+A comment, in comparison to a response, is a short note referring to either a post or comment. It is usually used for clarification or to request more information. 
+
+### Flags
+
+Flags send a user contribution to a moderation queue for review. Use these to report material inappropriate for a particular site, or anything that breaks the Code of Conduct or Terms of Service. 
+
+### Upvotes
+
+Upvotes help a post or answer gain visibility and act as an indicator of how the community values the contribution. They are also used in combination with downvotes to determine if it is well-received. 
+
+### Suggest edits
+
+Sometimes a user's contribution could use a little help. This could be due to grammar and formatting issues, or maybe the original poster has mentioned important information in a comment and needs to add it to the original post. Suggesting edits helps improve the quality of a site's content.
+
+### Suggest duplicates
+
+You may not have been the first person to ask this question! Duplicates aren't necessarily a bad thing, but for people looking for information it's helpful to have links to any previous posts on the same topic. 
+
+### Downvotes
+
+Downvotes are the negative analogue to upvotes. They are used to reduce the visibility of low quality contributions and are used in the calculation of whether or not a particular contribution is well-received. 
+
+### Vote-to-lock
+
+Some contributions need serious help. They might be off-topic, or unclear without additional information from the original poster. In these cases the best thing to do is often to lock the post so it's no longer able to accept responses until the original poster updates his work. All lock votes are based on certain categories, each associated with a linked community Meta post. A certain threshold of lock votes vs. no-lock votes must be met before a post is locked, and editing the post to adhere to site guidelines will allow it to be unlocked. 
```

Comment:
There are not necessarily linked community meta posts.  This might be getting a little too detailed for this list.  We do want to convey that we're waiting for an edit from the OP, so do keep that.